feast_trino/trino.py

```diff
@@ -37,6 +38,15 @@ class TrinoOfflineStoreConfig(FeastConfigBaseModel):
     catalog: StrictStr
     """ Catalog of the Trino cluster """
 
+    connector: Dict[str, Any]
+    """
+    Trino connector to use as well as potential extra parameters.
+    Needs to contain at least the path, for example
+    {"type": "bigquery"}
+    or
+    {"type": "hive", "file_format": "parquet"}
+    """
+
```

Comment:
I'm not an expert on Pydantic (yet 😄 ), but I believe it supports [sub-models for settings management](https://pydantic-docs.helpmanual.io/usage/settings/). Maybe you should give that a go?

Reply:
Not an expert yet. I would rather merge this PR for now and get back to it. This is a good point 👍